doc/source/blog_posts/2022/TRACER.ipynb

```diff
@@ -52,525 +53,36 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 15,
+   "execution_count": null,
    "id": "9d9e7920-e4d3-4403-8203-036d14ff0f67",
    "metadata": {},
-   "outputs": [
```

Comment:
Is there a reason why its deleting alot from this notebooks?

Reply:
That's a feature of nbstripout. It removes all output from notebooks and only keeps the minimal notebook source in the git.

Only makes sense, if the output is regenerated during CI for whatever reason (eg. Docs, Blogpost etc) 8t is needed.

Reply:
Ah gotcha makes sense! Thanks @kmuehlbauer !

Reply:
Yeah - with the blog posts, I do not think we would want this behavior... I can remove the nbstripout check